core/vm/logger_trace.go

```diff
@@ -0,0 +1,126 @@
+package vm
+
+import (
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/common/hexutil"
+	"github.com/scroll-tech/go-ethereum/core/types"
+)
+
+type traceFunc func(l *StructLogger, scope *ScopeContext, extraData *types.ExtraData) error
+
+var (
+	// OpcodeExecs the map to load opcodes' trace funcs.
+	OpcodeExecs = map[OpCode][]traceFunc{
+		CALL:         {traceToAddressCodeHash, traceLastNAddressCodeHash(1), traceOriginProof, traceLastNAddressProof(1)},
+		CALLCODE:     {traceToAddressCodeHash, traceLastNAddressCodeHash(1), traceOriginProof, traceLastNAddressProof(1)},
+		DELEGATECALL: {traceToAddressCodeHash, traceLastNAddressCodeHash(1)},
+		STATICCALL:   {traceToAddressCodeHash, traceLastNAddressCodeHash(1)},
+		CREATE:       {traceSenderAddress, traceNonce},
+		CREATE2:      {traceSenderAddress},
```

Comment:
And also sorry that, I made mistake in our [doc](https://www.notion.so/EVM-Trace-2e476641b6484b71a029c868516ad7b2): 
here for `CREATE` and `CREATE2` we need "callerAddress" "callerNnoce" instead of "senderAddress" "senderNonce".

we can use "ScopeContext.Contract.CallerAddress" to retrieve caller_address

Reply:
I also just update our doc